src/services/token.service.js

```diff
@@ -0,0 +1,29 @@
+const { Token } = require('../models/Token.model');
+
+const save = async (userId, newToken) => {
+  const token = await Token.findOne({ where: { UserId: userId } });
```

Comment:
Consider adding error handling for the database operations to catch and log any errors that occur during the token retrieval process.